lib/python-beta/src/bailo/bailo.py

```diff
@@ -50,3 +73,106 @@ def find_models(
                 "search": search,
             },
         ).json()
+
+    def get_model(
+        self,
+        model_id: str,
+    ):
+        """
+        Retrieves a specific model using its unique ID.
+
+        :param model_id: Unique model ID
+        :return: JSON response object
+        """
+        return self.agent.get(
+            f"{self.url}/v2/model/{model_id}",
+        ).json()
+
+    def update_model(
+        self,
+        model_id: str,
+        name: Optional[str] = None,
```

Comment:
Name, description and visibility could be provided as a dictionary, which would save you from a lot of the repetitive content below.